test/run-tests

```diff
@@ -23,16 +23,64 @@ import urllib.parse
 import urllib.request
 
 import requests
+import cachecontrol
 import yaml
 
 HOSTNAME = socket.gethostname()
 
+transient_error_wait_time_default = 30
```

Comment:
constants are written in all-caps by convention in Python, see for example `HOSTNAME`.